tests/EngineTest.php

```diff
@@ -751,8 +751,14 @@ public function testContainerDicePdoWrapperTestBadParams() {
         $engine->route('/container', Container::class.'->testThePdoWrapper');
         $engine->request()->url = '/container';
 
-        $this->expectException(ErrorException::class);
-        $this->expectExceptionMessageMatches("/Passing null to parameter/");
+        // php 7.4 will throw a PDO exception, but php 8 will throw an ErrorException
+        if(version_compare(PHP_VERSION, '8.0.0', '<')) {
```

Comment:
This is highly unusual, but the behavior in PDO actually changed inbetween 7.4 and 8.0 to throw different error messages.